File: 백준/Bronze/21866. 추첨을 통해 커피를 받자/추첨을 통해 커피를 받자.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    vector<int> arr(9),maxn={100,100,200,200,300,300,400,400,500};
    int hac=0,total=0;
    for(int i=0;i<9;i++) {
        cin>>arr[i];
        if(arr[i]>maxn[i]) {
            hac=1;
            break;
        }
        total+=arr[i];
    }
    cout<<(hac?"hacker":(total>=100?"draw":"none"));
    return 0;
}

File: 백준/Bronze/21866. 추첨을 통해 커피를 받자/추첨을 통해 커피를 받자.py
import sys
input=lambda:sys.stdin.readline().rstrip()
maxn=[100,100,200,200,300,300,400,400,500]
arr=[*map(int,input().split())]
hac,total=0,0
for i in range(9):
    if arr[i]>maxn[i]:
        hac=1
        break
    total+=arr[i]
print("hacker" if hac else "draw" if total>=100 else "none")